datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/jmx/{check_name}/setup.py

```diff
@@ -25,7 +25,7 @@ def get_dependencies():
         return f.readlines()
 
 
-CHECKS_BASE_REQ = 'datadog-checks-base'
+CHECKS_BASE_REQ = 'datadog-checks-base>=23.2.0'
```

Comment:
```suggestion
CHECKS_BASE_REQ = 'datadog-checks-base>=23.6.0'
```
Shouldn't it be `23.6.0` ? https://github.com/DataDog/integrations-core/blob/7.33.x/datadog_checks_base/CHANGELOG.md